mails/themes/classic/components/footer.html.twig

```diff
@@ -3,6 +3,6 @@
 </tr>
 <tr>
   <td class="footer" style="border-top: 4px solid #333333">
-    <span>{{ '<a href="{shop_url}">{shop_name}</a> powered by <a href="http://www.prestashop.com/">PrestaShop™</a>'|trans({}, 'Emails.Body', locale)|raw }}</span>
+    <span>{{ '<a href="{shop_url}">{shop_name}</a> powered by <a href="https://www.prestashop.com/?utm_source=marchandprestashop&utm_medium=e-mail=utm_campaign=footer_1-7">PrestaShop™</a>'|trans({}, 'Emails.Body', locale)|raw }}</span>
```

Comment:
Couldn't it be a dynamic URL? In my opinion, this should not be an element of translation.

Reply:
Not fully dynamic (the url will be static in the template) But yes I can modify it so that at least it doesn't appear in crowdin